pkgs/tools/misc/tmux/default.nix

```diff
@@ -63,8 +63,20 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   postInstall = ''
-    mkdir -p $out/share/bash-completion/completions
+    mkdir -p $out/share/bash-completion/completions $out/nix-support
     cp -v ${bashCompletion}/completions/tmux $out/share/bash-completion/completions/tmux
+    ${if stdenv.isDarwin then ''
+      mkdir -p $terminfo/share/terminfo/74
+      cp -v ${ncurses}/share/terminfo/74/tmux $terminfo/share/terminfo/74
+      # macOS ships an old version (5.7) of ncurses which does not include tmux-256color so we need to provide it from our ncurses.
+      # However, due to a bug in ncurses 5.7, we need to first patch the terminfo before we can use it with macOS.
+      # https://gpanders.com/blog/the-definitive-guide-to-using-tmux-256color-on-macos/
+      tic -o $terminfo/share/terminfo -x <(TERMINFO_DIRS=${ncurses}/share/terminfo infocmp -x tmux-256color | sed 's|pairs#0x10000|pairs#32767|')
+    '' else ''
+      mkdir -p $terminfo/share/terminfo/t
+      cp -v ${ncurses}/share/terminfo/t/tmux* $terminfo/share/terminfo/t
```

Comment:
Since tmux already depends on ncurses I'm a bit reluctant to just copy and duplicate the terminfo files in a new tmux output. Symlinks would also work here, right? Could we maybe only add the terminfo output on darwin since it doesn't really make sense on other platforms where we have our version of ncurses?

Reply:
Updated :+1: